packages/mobile/src/send/SendAmount.tsx

```diff
@@ -225,7 +224,11 @@ function SendAmount(props: Props) {
 
     dispatch(hideAlert())
 
-    if (addressValidationType !== AddressValidationType.NONE) {
+    if (
```

Comment:
Extra logic to ensure that secure send is not triggered for QR code or direct address transactions